src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@carlosmiranda Sorry, I have not understood you.
If i remove the `@Ignore` for the test `addressesInHeadersAddedWithoutSlashes` this test will break.
So we have to wait for the new puzzle I create in `RqLive` to finish this task?